packages/app/src/containers/LayerList.tsx

```diff
@@ -207,9 +207,27 @@ export default memo(function LayerList() {
     selectedObjects.length === 1 &&
     items.find((i) => i.id === selectedObjects[0])?.type === 'symbolInstance';
 
+  const canBeSymbol = useMemo(() => {
+    const selectedItems = items.filter((i) => selectedObjects.includes(i.id));
+
+    return (
+      selectedItems.length >= 1 &&
+      !selectedItems.some((l) => l.type === 'symbolMaster') &&
+      ((selectedItems[0].type === 'artboard' &&
+        selectedItems.every((l) => l.type === 'artboard')) ||
```

Comment:
If we're already checking that `every` item is an artboard, can we skip checking the first item, `selectedItems[0].type === 'artboard'`?